content/articles/how-to-design-infinite-impulse-response-filters-using-matlab/index.md

```diff
@@ -0,0 +1,84 @@
+### HOW TO DESIGN AN INFINITE IMPULSE RESPONSE FILTERS(IIR)
+### Introduction
+To filter is to remove the unwanted properties of a signal. IIR filters with an infinite number of impulses. Here the output y(n) response depends on the present input x(n), previous input x(n-1) as well as the previous output y(n-1). The differential equation for the IIR filter can be given by the differential equation $y(n)=b_0x(n)+b_1x(n-1)+---b_mx(n-m)-a_1y(n-1)---a_ny(m-n)$. To see more about this, you can get it [here](https://www.sciencedirect.com/topics/engineering/iir-filters). Since the filter uses the previous outputs, there is feedback from the filter structure. The design of an IIR filter is based on the transfer function($H_s$) that satisfies the requirement of the filter requirement. The transfer function($H_s$) is the ratio of the output response to the input response.
```

Comment:
A great intro here, but could you twist it a little: by adding the following:    

-  Tell us what's an IIR in detail,for example`The infinite impulse response (IIR) filter is a recursive filter in that the output from the filter is computed by using the current and previous inputs and previous outputs`.  This has more meaning than the one you have provided!